### PR TITLE
docs: custom field tutorial and reorg

### PR DESCRIPTION
**Pull request by tomkit**
- reorganize sidebar to classify diff tutorials
- add a basic custom field tutorial

Screenshot 2023-10-30 at 4 34 52 PM

## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

**Review by lucasmarshall — APPROVED** [review]

File: docs/docs/tutorials/custom-fields.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+- Hubspot
+
+## Create a custom field
+
+Use Supaglue's [Metadata (Create Property) API](https://docs.supaglue.com/api/v2/metadata/create-property) to define a custom property on the Hubspot Contact object.
+
+The curl will look like the following:
+
+```curl
+curl --location 'https://api.supaglue.io/metadata/v2/properties/contact' \
```
Comment (asdfryan) [inline docs/docs/tutorials/custom-fields.md:25]: this needs to be changed to `https://api.supaglue.io/crm/v2/metadata/properties/contact`

File: docs/docs/tutorials/custom-fields.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+This tutorial assumes you have gone through Supaglue's [Quickstart](../quickstart) and will use the following technologies:
+
+- HTTP
+- curl
+- Hubspot
+
+## Create a custom field
+
+Use Supaglue's [Metadata (Create Property) API](https://docs.supaglue.com/api/v2/metadata/create-property) to define a custom property on the Hubspot Contact object.
```
Comment (asdfryan) [inline docs/docs/tutorials/custom-fields.md:20]: This link needs to be changed

Comment (asdfryan) [reply docs/docs/tutorials/custom-fields.md]: I went ahead and updated

File: docs/docs/tutorials/custom-fields.md (hunk 1 above)
Comment (asdfryan) [reply docs/docs/tutorials/custom-fields.md]: I went ahead and updated

**Pull request merged**